docs/core/extensions/cloud-service.md

```diff
@@ -212,32 +262,76 @@ Expand the **REGISTRIES** node, select **Azure**, your  subscription > the conta
 :::image type="content" source="media/vs-code-deploy-to-aci.png" alt-text="Visual Studio Code - Docker: Deploy image to Azure Container Instances":::
 
 :::zone-end
+:::zone target="docs" pivot="cli"
+
+To create a container instance, you'll need to create a container group using the [`az container create`](/cli/azure/container#az_container_create) command.
+
+```azurecli
+az container create -g <resource group> \
+  --name <instance name> \
+  --image <registry name>.azurecr.io/<image name>:latest \
+  --registry-password <password>
+```
+
+Provide the appropriate values:
 
-:::zone target="docs" pivot="visualstudio,cli"
+- `<resource group>`: the resource group name that you've been using in this tutorial.
+- `<instance name>`: the name of the container instance.
+- `<registry name>`: the name of the container registry.
+- `<image name>`: the name of the image.
+- `<password>`: the password to the container registry &mdash; you can get this from the Azure Portal, Container Registry resource > **Access Keys**.
+
+:::zone-end
+:::zone target="docs" pivot="visualstudio"
 
 To create a container instance, you'll need to [create a new resource](https://ms.portal.azure.com/#create/Microsoft.ContainerInstances) in the Azure portal.
 
 1. Select the same **Subscription**, and corresponding **Resource group** from the previous section.
 1. Enter a **Container name** &mdash; `appcloudservice-container`.
 1. Select a **Region** that corresponds to the previous **Location** selection.
-1. Select **Azure Container Registry** as the **Image source**.
+1. For **Image source**, select **Azure Container Registry**.
 1. Select the **Registry** by the name provided in the previous step.
 1. Select the **Image** and **Image tag**.
 1. Select **Review + create**.
 1. Assuming **Validation passed**, select **Create**.
 
+It may take a moment for the resources to be created, once created select the **Go to resource** button.
+
 For more information, see [Quickstart: Create an Azure container instance](/azure/container-instances/container-instances-quickstart-portal).
```

Comment:
If possible, i think it'd be good for you to provide folks with the CLI command required to enlist their Azure Sub to be able to use ACI. i don't know if ACI is one of the automatically-enabled providers; in a lot of scenarios where I'm testing this scenario in new Azure subs, I've had to enable the ACI RP. If that's lacking from the article folks may get lost once they try to run their container image as an instance. Small fries, probably, and I'm sure the ACI docs you link off to would suffice.

Reply:
I added all the AZ CLI commands, and tested them. This should be golden now.